packages/element-library/src/test/index.js

```diff
@@ -13,19 +13,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 /**
- * Internal dependencies
+ * External dependencies
  */
-import { TEXT_ELEMENT_DEFAULT_FONT } from '../../app/font/defaultFonts';
 import {
   createNewElement,
   createPage,
   duplicatePage,
   duplicateElement,
-} from '..';
+  registerElementType,
+} from '@googleforcreators/elements';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '@googleforcreators/design-system';
```

Comment:
Yeah so to me this feels like this constant belongs in `elements` not `design-system` since it's the default for an element that the design system should be agnostic about.

Reply:
Thank you! Moved in https://github.com/GoogleForCreators/web-stories-wp/pull/10452/commits/fad8ae498113d54de0b225d524d06c78404c9f6b